lib/ops/resources/resources.go

```diff
@@ -76,6 +87,11 @@ func (r ListRequest) Check() error {
 	if r.Kind == "" {
 		return trace.BadParameter("resource kind is mandatory")
 	}
+	kind := modules.Get().CanonicalKind(r.Kind)
+	resources := modules.Get().SupportedResourcesToRemove()
```

Comment:
Why does "list" request check supported resources to "remove"?

Reply:
Thanks - I already spotted it.